data/transport missions.txt

```diff
@@ -1868,6 +1872,309 @@ mission "Terraforming 5"
 
 
 
+event "terraforming Rand"
+	planet Rand
+		description `Rand is a desert world, too dry for much farming and with gravity low enough to be uncomfortable for most human beings. It is, however, the best source of heavy metals in the galactic south. Aside from the managers of the mining companies, nearly all the people here are migrant workers from elsewhere in the Dirt Belt, who have come to spend a season working for the relatively high wages that uranium mining offers, either to send money off-world or to save it up in order to build a better life for themselves.`
+		description `	Recent terraforming experiments have caused an increase in precipitation. Flash floods have scoured the landscape in some areas, but sandstorms are also less frequent, and the locals say the heat is a bit less oppressive than it used to be.`
+
+
+
+mission "terraforming timer"
+	landing
+	invisible
+	to offer
+		has "Terraforming 5: done"
+	on offer
+		event "terraforming timer" 4 8
+		fail
+
+event "terraforming timer"
+
+mission "Terraforming 6"
+	landing
+	name "Unlawful Use of an Asteroid"
+	description `Travel to <destination> by <date> for questioning on your "unlawful use of an asteroid."`
+	source
+		government "Republic"
+		not attributes "deep"
+	destination "Earth"
+	deadline
+	to offer
+		has "event: terraforming timer"
+		or
+			has "main plot completed"
+			and
+				not "event: pug invasion"
+				not "event: fwc pug invasion"
+	
+	on offer
+		conversation
+			`When you land on <origin> and exit your ship, you are stopped by a Navy officer with a group of guards. "Captain <last>. You are wanted for..." the officer pauses and looks down at the tablet in his hand. "The 'unlawful use of an asteroid.' You are to travel to <destination> immediately for questioning."`
+			choice
+				`	"Understood, officer."`
+					accept
+				`	"I'm wanted for what?"`
+				`	(Run away.)`
+					flee
+			
+			`	"The unlawful use of an asteroid, Captain. I'm not entirely sure what that means, but I've been told to send you to <planet>. Your compliance would be much appreciated."`
+			choice
+				`	"Alright. Hopefully I can get an answers as to what this is about."`
+					accept
+				`	(Run away.)`
+					flee
+			
+	on decline
+		"reputation: Republic" <?= -10
+	on fail
+		"reputation: Republic" <?= -10
+	on complete
+		conversation
+			`You are quickly escorted from the crowded <planet> spaceport to a court building in a wealthy district of the city and asked multiple questions pertaining to when you crashed an asteroid into Rand. "Why did you purposefully do crash an asteroid into a planet? What did you expect to happen? Do you know how many people could have died?" You answer the questions to the best of your ability, and after roughly an hour of questioning you are transferred to a different room where you find that Eric, Alaric, and Amy are all here as well.`
```

Comment:
`pertaining to when you crashed an asteroid into Rand.`
Needs a rephrase. Possibly `about the events on Rand.`

`purposefully do crash` ~~do~~

`You answer the questions to the best` ~~the questions~~

`where you find that Eric, Alaric, and Amy are all here as well.`  -> `where you find Eric, Alaric, and Amy.`

For dramatic effect, all characters could be handcuffed (lasercuffed? future police tech is probably sweet 💯 )